dll/win32/comctl32/comctl32.rc

```diff
@@ -0,0 +1,172 @@
+/*
```

Comment:
I think this file has to be replaced by a synced up version of
https://git.reactos.org/?p=reactos.git;a=blob;f=dll/win32/comctl32/rsrc.rc;h=7779c8ec1fcd5818bb911ffd3024c9bea2165537;hb=HEAD
because as you can see here, your new comctl32.rc file is a template file, and the english strings have to be translated.
I know that @AmineKhaldi did this properly in the past by running the po file translator and this allowed to get the different languages.

Reply:
Yeah im totally lost as to how this is suppose to work

Reply:
I did set this mechanism up now, so as we for further and further in sync i can do this similar to how amine did it in the past i believe.